_data/projects/raiden.yml

```
@@ -10,7 +10,7 @@ tags:
 - state-channels
 upforgrabs:
   name: good first issue
```

And this tag now needs to be updated:

```suggestion
  name: "Tag / Good First Issue"
```